workflows/run_gpu_mask.lsf

```diff
@@ -8,14 +8,26 @@
 #BSUB -e /data/pnl/projects/tmp/gpu-mask-%J-%I.err
 #BSUB -n 4
 
+# a single caseid or a text file with list of cases
```

Comment:
It may be useful to include the following comment: "Use of caselist recommended if submitting job, but not if running directly on terminal."